services/docker-compose.yml

```diff
@@ -35,6 +35,8 @@ services:
       - LOG_LEVEL=${LOG_LEVEL:-WARNING}
     env_file:
       - ../.env
+    volumes:
```

Comment:
If you do this, and I think it is necessary, please make sure that this service is properly isolated. It should not be on a network with other plattform services. There should be no way to execute code inside this container or to manipulate the container via REST requests

Reply:
Good point. I think we should do that systematically by having some protocols or tools to ensure that. It is very easy to fall into that situation while developing a feature if you do not have any sort of tool that alerts about it.